### PR TITLE
Fix RestFul API

### PR DESCRIPTION
**Pull request by wbloszyk**
## Subject

I am targeting this branch, because this change respect BC.

Part of https://github.com/sonata-project/dev-kit/issues/778

## Changelog

```markdown
### Added
- Added public alias `Sonata\ClassificationBundle\Controller\Api\CategoryController` for `sonata.classification.controller.api.category` service
- Added public alias `Sonata\ClassificationBundle\Controller\Api\CollectionController` for `sonata.classification.controller.api.collection` service
- Added public alias `Sonata\ClassificationBundle\Controller\Api\TagController` for `sonata.classification.controller.api.tag` service
- Added public alias `Sonata\ClassificationBundle\Controller\Api\ContextController` for `sonata.classification.controller.api.context` service
### Fixed
- fix RestFul API - `Class could not be determined for Controller identified` Error
```

### CODE REVIEW

File: src/Resources/config/routing/api.xml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <routes xmlns="http://friendsofsymfony.github.com/schema/rest" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://friendsofsymfony.github.com/schema/rest https://raw.github.com/FriendsOfSymfony/FOSRestBundle/master/Resources/config/schema/routing/rest_routing-1.0.xsd">
```
Comment (jordisala1991) [inline src/Resources/config/routing/api.xml:2]: This file in the @phansis pr was handled different

Comment (wbloszyk) [reply src/Resources/config/routing/api.xml:2]: Yes, but i dont add support for fos/rest v4

**Comment by wbloszyk** [comment]
@phansys Tomorrow I will open `fix restful api` PRs for all bundles where is it require. Additional I will open PR in sandbox for check it and fix on sandbox.

IMHO Functional tests should be add for all bunsles. Some things like router config should be tested. 

BTW
I think about move fixtures from sandbox to seperate bundle. This will allow install demo data for each bundles and allow test it in Functional tests. I will open dev-kit issue for it.

**Comment by wbloszyk** [comment]
@phansys Can we merge it without tests? I think we should first decide what to do with RESTful. 
https://github.com/sonata-project/dev-kit/issues/778#issuecomment-653759983

**Comment by phansys** [comment]
>@phansys Can we merge it without tests? I think we should first decide what to do with RESTful.
sonata-project/dev-kit#778 (comment)

No problem.

**Comment by jordisala1991** [comment]
Thank you @wbloszyk

**Pull request merged**